src/definition/accessors/ISchedulerExtend.ts

```diff
@@ -5,6 +5,7 @@ export interface ISchedulerExtend {
      * Register processors that can be scheduled to run
      *
      * @param {Array<IProcessor>} processors An array of processors
+     * @returns List of task ids run at startup, or void no startup run is set
      */
-    registerProcessors(processors: Array<IProcessor>): Promise<void>;
+    registerProcessors(processors: Array<IProcessor>): Promise<void | Array<string>>;
```

Comment:
The result list has the same order as the processors with startup settings.

Like this: the register list has the following processors (void has no startup setting, id has startup setting)

```
- void
- id1
- id2
- void
- void
- id3
```


Yields the following list as result 

```
- id1
- id2
- id3
```

I think for that new record structure to have meaning, the "key" part should be a custom id set by the developer/user that _could_ fallback to the processor id if it isn't set. The alternative would be to put the processor id by default and this could possibly return lots of the same string (processor id) in the list, _kind of_ putting us in the same situation. We would have differentiation between tasks created from different processors, but if there are multiple tasks using the same processor we wouldn't know who's who